### PR TITLE
Feature/readme

### PR DESCRIPTION
**Pull request by hayleyw7**
# Thank you for pushing your code!
## What (if any) features are you implementing?
- Create and populate file
- this PR closes #21 
## What (if anything) did you refactor?
- n/a
## Were there any issues that arose?
- n/a
## Is there anything that you need from your teammate?
- n/a
## Any other comments, questions, or concerns?
- I'll add a gif once we are 100% sure that everything else is complete.
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?
- [ ] open index.html
- [ ] console.log()
- [ ] dev tools

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,94 @@
-# Refactor Tractor (Fitlit Version B)
+# Refactor Tractor: FitLit (B)
 
-## Setup
+This is a project where we were required to refactor, navigate, and build on top of somebody else’s pre-existing codebase. Since working on another's code is common in software development, it was our first big opportunity to get comfortable doing this.
```
Comment (williamphelps13) [inline README.md:3]: Good Summary!

File: README.md [hunk 2]
```diff
@@ -1,19 +1,94 @@
...
+## Reflections
 
-Your linter will look only at the JavaScript files you have within the `src` and the `test` directories.
+Our biggest challenges were navigating the functions and organization of another's code and making network requests to API endpoints to retrieve and manipulate data.
+
+Big successes were implementation of SASS for the first time, getting accessibility to a score of 100 on Lighthouse, and resolving all of the above challenges with passing tests.
+
+## Languages and technology
+
+Written in JavaScript, Mocha, Chai, HTML, & SCSS
```
Comment (williamphelps13) [inline README.md:21]: Good full list!

**Review by MTCawthray — COMMENTED** [review]
Nice lookin' read me!

**Review by williamphelps13 — APPROVED** [review]
Looks good Hayley! We'll add that gif tomorrow!

**Pull request merged**